session-replay/Sources/DatadogSessionReplay/Processor/SRDataModelsBuilder/WireframesBuilder.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal typealias Wireframe = SRMobileFullSnapshotRecord.Data.Wireframes
```

Comment:
Sorry, I should have better annotate this in the code. We can't make it `private` as it is used in `RecordsBuilder`. The "wireframe" type will be commonly referenced in different components of `Processor`, so I wanted to add more convenience to this type name. The idea I had was to first add `typealias` and then solve the problem at its origin, which is in `rum-models-generator`. I created `RUMM-2440` to enhance [SR code decorator](https://github.com/DataDog/dd-sdk-ios/blob/develop/tools/rum-models-generator/Sources/CodeDecoration/SRCodeDecorator.swift), generate `SRWireframe` directly and get rid of this (and few other) aliases. How does it sound?
```swift
/// TODO: RUMM-2440 - configure models generator to emit root-level `SRWireframe` instead of this
internal typealias SRWireframe = SRMobileFullSnapshotRecord.Data.Wireframes
```

Reply:
Sounds good 👍